Add Powershell Installation Instructions for Copilot (#52141)
Azure/azure-sdk-for-net: .github/copilot-instructions.md

This repo contains Azure SDKs for various Azure services and tools that generate them.

## Prerequisites

> - To use MCP tool calls, user must have PowerShell installed. Provide [PowerShell installation instructions](https://learn.microsoft.com/powershell/scripting/install/installing-powershell?view=powershell-7.5) if not installed, and recommend restarting VSCode to start the MCP server.

Reviewer: ```suggestion
- To use MCP tool calls, PowerShell must be installed. Provide [PowerShell installation instructions](https://learn.microsoft.com/powershell/scripting/install/installing-powershell) if not installed, and recommend restarting the IDE to use the MCP server.
```

I don't think we meant to bind this to a specific version.   Most .NET developers do not use VSCode, so I'd like to make that more generic.
